instrumentation/spring/README.md

```diff
@@ -841,14 +837,14 @@ To configure OpenTelemetry tracing with the OTLP, Zipkin, or Jaeger span exporte
     <groupId>io.opentelemetry.instrumentation</groupId>
     <artifactId>opentelemetry-zipkin-spring-boot-starter</artifactId>
     <version>OPENTELEMETRY_VERSION</version>
-</dependency>
+</dependency> 
```

Comment:
do you know why the trailing space is relevant?

Reply:
it looks like the spotless check is failing on the trailing space

Reply:
fixed